auction-app-backend/src/main/java/com/example/auctionapp/model/Product.java

```diff
@@ -0,0 +1,96 @@
+package com.example.auctionapp.model;
+
+import com.example.auctionapp.entity.CategoryEntity;
+import com.example.auctionapp.entity.ProductEntity;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+public class Product {
+
+    private UUID id;
+    private String name;
+    private String description;
+    private BigDecimal startPrice;
+    private LocalDateTime startDate;
+    private LocalDateTime endDate;
+    private String imageUrl;
+    private String status;
+    private UUID categoryId;
+
+    public Product() {
+    }
+
+    public UUID getId() {
+        return id;
+    }
+
+    public void setId(UUID id) {
```

Comment:
One thing I've forgot to mention here, use final for those parameters 
`final UUID id`